managers/opportunity/index.js

```diff
@@ -41,20 +41,139 @@ class opportunityManager {
     }
   }
 
+  // http://localhost:3030/opportunity?limit=4&page=2, Example of how to use the Get Method
+
+  //  page => Means the Current Page that the User Wants
+  //  limit => The count of pages that should be rendered starting from the given page
```

Comment:
limit - The count of maximum opportunities returned on 1 page. 

The comments should be proper so that no one is misled by them.

Reply:
oh yess, my mistake I left this out from the previous code , will correct it